.storybook/main.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   stories: [
     "./docs/stories/Welcome.stories.mdx",
     "./docs/stories/Colors.stories.mdx",
+    "./docs/stories/Typography.stories.mdx",
```

Comment:
You have to pass them individually if you want a specific order. Otherwise, it will take those and alphabetize them - which in our case would work actually with the stories we currently have but I went with the explicit route here.